packages/gasket-plugin-command/test/command.test.js

```diff
@@ -83,28 +80,30 @@ describe('GasketCommand', function () {
   beforeEach((done) => {
     process.env.NODE_ENV = 'development';
 
-    gasket = { exec: sinon.spy() };
-    PluginEngine = sinon.stub().returns(gasket);
+    gasket = { exec: jest.fn() };
+    MockPluginEngine = jest.fn().mockReturnValue(gasket);
 
-    GasketCommand = proxyquire('../lib/command', {
-      '@gasket/engine': PluginEngine
-    });
+    jest.mock('@gasket/engine', () => MockPluginEngine);
+    GasketCommand = require('../lib/command');
+    // GasketCommand = proxyquire('../lib/command', {
+    //   '@gasket/engine': MockPluginEngine
+    // });
```

Comment:
Ready to remove commented code?
```suggestion
    // GasketCommand = proxyquire('../lib/command', {
    //   '@gasket/engine': MockPluginEngine
    // });
```